src/fromager/bootstrapper.py

```diff
@@ -323,6 +323,12 @@ def _download_wheel_from_cache(
             wheelfile_name = pathlib.Path(urlparse(wheel_url).path)
             pbi = self.ctx.package_build_info(req)
             expected_build_tag = pbi.build_tag(resolved_version)
+            # Log the expected build tag for debugging
+            logger.info(f"{req.name}: has expected build tag {expected_build_tag}")
+            # Get changelogs for debug info
+            changelogs = pbi.get_changelog(resolved_version)
+            logger.info(f"{req.name} has change logs {changelogs}")
```

Comment:
Logging at info level here is going to be pretty noisy. The thing we really care about at info level is the number (the expected build tag), which you're logging on 327. So maybe change this to a debug message.

Reply:
Okay, makes sense